config/agora_config.yaml

```diff
@@ -0,0 +1,86 @@
+################################################################################
```

Comment:
This file shouldn't be here.

Reply:
So when It run Stoa Docker, do It have to deliver it from the outside?

Reply:
Yes, see e.g. https://github.com/bpfkorea/agora/blob/e14004eefac4c68fda6127fcae2236bac52b101f/tests/system/docker-compose.yml#L17

Reply:
The file was changed to an example file.
Now refer to external config file
`docker run -p 127.0.0.1:3836:3836/tcp -v $(pwd)/config/:/config/config.example.yaml -e "CONFIG=config/config.example.yaml" bpfk/stoa`